web/src/narrow.js

```diff
@@ -209,6 +209,10 @@ export function activate(raw_terms, opts) {
         if (filter.has_operator("id")) {
             id_info.target_id = Number.parseInt(filter.operands("id")[0], 10);
         }
+        if (filter.has_operand("has","reaction") && !(filter.has_operator("sender")) ) {
+            console.log("Pushing sender to filter")
+            filter._terms.push({operator:"sender", operand:people.my_current_email(), negated:false})
```

Comment:
I don't understand this change; I think the idea was we'd just have a link to the `sender:me has:reaction` search query, not that we'd make it impossible to search for just `has:reaction`, as this proposes.

Reply:
It looks like you remove this in a future commit in this PR. In general, you should clean that sort of thing up before asking for review, as it can be confusing and waste reviewer time that would be better spent on changes that you think are correct.

Check out the [Zulip commit guidelines](https://zulip.readthedocs.io/en/latest/contributing/commit-discipline.html#commit-messages) and today's community event for more details.